config/initializers/cors.rb

```diff
@@ -9,7 +9,7 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins ENV['CLIENT_ORIGIN'] || 'http://localhost:5173'
+    origins ENV['CLIENT_ORIGIN'].presence || 'http://localhost:5173'
```

Comment:
This ensures that if the env variable is set to `''` the client origin is still a valid URI.